kong/plugins/aws-lambda/handler.lua

```diff
@@ -72,14 +123,15 @@ function AWSLambdaHandler:access(conf)
   if err then
     return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
   end
-
   -- Trigger request
   local client = http.new()
-  client:connect(host, 443)
   client:set_timeout(conf.timeout)
-  local ok, err = client:ssl_handshake()
-  if not ok then
-    return responses.send_HTTP_INTERNAL_SERVER_ERROR(err)
+  client:connect(host, (conf.aws_port or 443))
```

Comment:
In fact it's not mixing application and test code it's allowing to override default port and host which should be allowed anyway. Along with overriding default path "/2015-03-31/functions/%s/invocations"
There was logic in kong/plugins/aws-lambda/v4.lua that allowed to path port there 
`  local port = tbl.port or (tls and 443 or 80)
`
I only added possibility to pass host as well and make it configurable. I removed configuration for ssl in current version of PR